documentation/docs/02-template-syntax/02-basic-markup.md

```diff
@@ -29,7 +29,7 @@ By default, attributes work exactly like their HTML counterparts.
 As in HTML, values may be unquoted.
 
```

Comment:
```suggestion
<!-- prettier-ignore -->
```

Yeah, prettier will turn it back to quoted values if someone runs format on the file again.